src/LazyLoad.jsx

```diff
@@ -60,10 +64,11 @@ LazyLoad.propTypes = {
   children: PropTypes.node.isRequired,
   height: PropTypes.oneOfType([
     PropTypes.string,
-    PropTypes.number,
```

Comment:
lol, i know IE8 doesn't like it, not that i care about IE8 anymore but it's a habit :)